### DISCUSSION TITLE
How to update Terms and conditions on sunbird ed env

### DISCUSSION THREAD

**Discussion by ashokreddy1208** [Sunbird-Lern/Community#105] [Documentation] [answered]
Hi @reshmi-nair , We need to update Terms and conditions on sunbird ed staging env, please let me know the steps to do same..

New  T&C link:- https://sunbirdstagingpublic.blob.core.windows.net/privacy-policy/terms-of-use.html

cc @Hari-stackroute @rhwarrier @Krishnaj20

**Comment by reshmi-nair** [comment, marked as answer]
@ashokreddy1208 Please use the system settings API to update this. 
There are 4 types of t&c - 
tncConfig - for any user login
groupsTnc - while creating group
orgAdminTnc- For org admins
reportViewerTnc - for report viewer role for viewing reports

Please make sure you are updating for the correct config.

API endpoint : /api/data/v1/system/settings/set
 {
  "request":{
                "id": "reportViewerTnc",
                "field": "reportViewerTnc",
                "value": "{\"latestVersion\":\"4.0.0\",\"4.0.0\":{\"url\":\"https://sunbirdstagingpublic.blob.core.windows.net/termsandcondtions/terms-and-conditions-v9.html#administratorGuidelines\"}}"
            }
}
---------------------
{
  "request":{
                "id": "groupsTnc",
                "field": "groupsTnc",
                "value": "{\"latestVersion\":\"3.5.0\",\"3.5.0\":{\"url\":\"https://sunbirdstagingpublic.blob.core.windows.net/termsandcondtions/terms-and-conditions-v9.html#groupGuidelines\"}}"
           }
}
---------------------------
 {
  "request":{
                "id": "orgAdminTnc",
                "field": "orgAdminTnc",
                "value": "{\"latestVersion\":\"3.5.0\",\"3.5.0\":{\"url\":\"https://sunbirdstagingpublic.blob.core.windows.net/termsandcondtions/terms-and-conditions-v9.html#administratorGuidelines\"}}"
            }
}
-----------------------
 {
  "request":{
                "id": "tncConfig",
                "field": "tncConfig",
                "value": "{\"latestVersion\":\"v12\",\"v12\":{\"url\":\"https://obj.stage.sunbirded.org/termsandcondtions/terms-and-conditions-v12.html\"}}"
            }
}
